static/app/views/performance/newTraceDetails/traceModels/traceTreeNode/transactionNode.tsx

```diff
@@ -51,10 +50,26 @@ export class TransactionNode extends BaseNode<TraceTree.Transaction> {
       spanChildrenCount === undefined ? true : spanChildrenCount > 1;
 
     if (value) {
-      this.space = [
-        value.start_timestamp * 1e3,
-        (value.timestamp - value.start_timestamp) * 1e3,
-      ];
+      if (
```

Comment:
Adds the validation for `value.timestamp` used to initialize `TransactioNode.space[1]`. Safer, helps pass a bunch of tests in `trace.spec.tsx`